File: src/pages/exercises/basic-bmi/_codes/php/response/bmi.php
<?php

$height = 1.65;
$weight = 60.0;
$bmiValue = $weight / $height ** 2;
$bmiStatus;

if ($bmiValue < 18.5) {
  $bmiStatus = 'Underweight';
} elseif ($bmiValue <= 24.9) {
  $bmiStatus = 'Normal weight';
} elseif ($bmiValue <= 29.9) {
  $bmiStatus = 'Overweight';
} else {
  $bmiStatus = 'Obesity';
}

// echo 'BMI - value: '.$bmiValue.' status: '.$bmiStatus;
// echo "BMI - value: $bmiValue status: $bmiStatus";
echo "BMI: ${bmiValue}\nResult: ${bmiStatus}";
 

File: src/pages/exercises/basic-bmi/_codes/ruby/response/bmi.rb

weight = 60
height = 1.65
result = ''

bmi = weight / height ** 2

if bmi < 18.5
  result = 'Underweight'
elsif bmi <= 24.9
  result = 'Normal weight'
elsif bmi <= 29.9
  result = 'Overweight'
else
  result = 'Obesity'
end

puts "BMI: #{bmi}\nResult: #{result}"
